content/raw/general/onboarding.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Login & Onboarding spec"
+description: ""
+date: 2022-08-16T08:48:57+00:00
+lastmod: 2022-08-16T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "general"
+weight: 100
+toc: true
+---
+
+# Login & Onboarding
+
+## Motivation
+
+## Definitions
+
+- Touch ID - Electronic fingerprint recognition used by apple desktop devices
+
+## Designs
+
+- [First time opening app](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1075%3A312335)
+- [Login](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1023%3A298171)
+- [New user](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1027%3A302661)
+
+## Use Cases
+
+## Functional Requirements
+
+### Initial Screen
+
+- If and only if it's the first time opening the app then the Initial Screen MUST BE displayed
+- The allow notifications screen MUST be displayed
+- The user MUST agree to the terms to proced
+    - `I aknowledge that Status Desktop is in Beta and by using it, I take the full responsibility for all risks concerning my data and funds.`
+    - Terms of Use
+    - Privacy Policy
+- The following two options should be shown
+    - "I'm new to Status"
+    - "I already use Status"
+
+### Login & Accounts
+
+#### Accounts and types of Logins
+
+- Each account MUST display:
+    - The profile picture
+        - If no profile picture is available then the initials matching the username MUST be displayed
```

Comment:
@iurimatias if no profile picture is available then two letters initials matching display name must be displayed, not username.